src/test/java/org/prebid/server/bidder/eplanning/EplanningBidderTest.java

```diff
@@ -250,6 +252,94 @@ public void makeHttpRequestsShouldSetCorrectUriWithSizeString() {
                         + "300x200");
     }
 
+    @Test
+    public void makeHttpRequestsShouldSetCorrectUriWithSizeStringFromFormatForMobile() {
+        // given
+        final BidRequest bidRequest = givenBidRequest(
+                bidRequestBuilder -> bidRequestBuilder.device(Device.builder().devicetype(1).build()),
+                impBuilder -> impBuilder
+                        .banner(Banner.builder()
+                                .format(asList(Format.builder().w(300).h(50).build(),
+                                        Format.builder().w(320).h(50).build()))
+                                .build()));
+
+        // when
+        final Result<List<HttpRequest<Void>>> result = eplanningBidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).hasSize(1)
+                .extracting(HttpRequest::getUri)
+                .containsOnly("https://eplanning.com/clientId/1/FILE/ROS?r=pbs&ncb=1&ur=FILE&e=testadun_itco_de%3A"
```

Comment:
Can be used containsExactly() and then no .hasSize(1) needed

Reply:
I think it is much more easier  for reader to identify that there can be only one request by hasSize(1) than counting how many values were passed to containsExactly. But probably one hasSize() use for test dedicated to checking that bidder produces only one request is enough, and in all other places containsExactly can be used.